.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: python
```

Comment:
It would be nice to use github actions instead of travis, but we can leave that for later if you want to.

Reply:
especially since travis is not configured... I decided not to do this since it's new to me and as it is now, travis was not set up anyway. I can have a look if I have the time.